engine/pokemon/correct_nick_errors.asm

```diff
@@ -70,5 +70,5 @@ CorrectNickErrors::
 	db "<ROUTE>",  "<GREEN>" + 1
 	db "<ENEMY>",  "<ENEMY>" + 1
 	db "<MOM>",    "<TM>"    + 1
-	db "<ROCKET>", "┘"       + 1
+	db "<ROCKET>", " "
```

Comment:
`" "` has no relation other than being one after. If border tiles don't count as control characters the problem is the labelling.

Reply:
I guess border tiles can count as control characters, then.